src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  currentFilter: string,
+  onFilterChange?: (filter: string) => void
+};
+
+export const TodosFilter: React.FC<Props> = ({
+  currentFilter,
+  onFilterChange = () => {},
+}) => {
+  return (
+    <ul className="filters" data-cy="todosFilter">
+      <li>
+        <a
+          href="#/"
+          className={classNames({
+            selected: currentFilter === Filter.All,
+          })}
+          onClick={() => {
+            onFilterChange(Filter.All);
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here